middleware/middleware-1.3.16.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: /release-notes/middleware/1.3.16
+title: Version 1.3.16
+---
+
+# fiskaltrust.Middleware 1.3.16 (Germany)
+_March 12, 2021_
+
+We're happy to announce that the fiskaly SCU is now available in our production systems (please see our [blog post](https://fiskaltrust.de/news/aktueller-status-zur-cloud-tse-202103/) for more details). Additionally, we've added a new signature that contains the TSE certification ID and (if required) information about the state of the certification and operational environment regulations. Finally, this version contains some bug fixes and usability improvements for the Middleware.
+
+<div class="alert alert--warning" role="alert">Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.</div>
+
+## Production release: fiskaly SCU
+In close collaboration with our partners, we've released the fiskaly SCU to production. To fulfill the legal requirements, we followed fiskaly's advice and added the informational phrase _"TSE in Evaluierung"_ ("TSE in evaluation") into the newly added certification identification signature item (see below). Please keep in mind that the fiskaly TSE is still in evaluation, and switching to the certified state will most likely require an update of the SCU. Please refer to our [blog post about the state of cloud TSEs](https://fiskaltrust.de/news/aktueller-status-zur-cloud-tse-202103/) for more details.
+
+## New signature item: Certification identification
+We've added an additional signature item to the receipt response of POS receipts that contains the certification identification of the used TSE (e.g. `BSI-K-TR1234-5677`), and additionally contains a hint about the current state of the certification process or the operational environment. The _ftSignatureType_ of the newly added item is `0x4445000000000022`, more information can be found [here](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/reference-tables/ftsignaturetype).
+
+The signature item was added for all TSEs, including hardware devices. Currently, additional information is displayed for the three cloud SCUs that are available in our production systems:
+
+| **SCU**                                      | **Certification identification**    |
```

Comment:
Should we add the certifications for the other TSEs to that table too?

Reply:
I'd rather not to highlight the point that those are the "important" ones to print onto the receipt, and added another hint that this is mandatory below